server/server.js

```diff
@@ -10,17 +10,19 @@ app.listen(PORT, () => {
 
 app.use(express.static(__dirname + '/../client/dist'));
 app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.urlencoded({ extended: true}));
 
 
-app.get('/book/:campId', async (req, res) => {
-  console.log('req.params.campId: ', req.params.campId);
-    await db.Booking.find({campId: 0})
-    .then((site) => {
-      console.log('site: ', site);
-      res.status(200).send(site);
-    })
-    .catch((err) => {
-      throw err;
-    });
-});
\ No newline at end of file
+app.get('/booking', async (req, res) => {
+  let campId = req.query.campId;
+  let numbCampId = Number(campId);
+  await db.Booking.find({campId: numbCampId})
+  .then((site) => {
+    res.send(site);
+  })
+  .catch((err) => {
+    throw err;
```

Comment:
consider sending back a response to the site/app so that its not just waiting for a response but it mknows there was an error somewhere.